src/index.css

```diff
@@ -488,6 +494,10 @@ button.shadow-box-title {
     justify-content: space-around;
 }
 
+#close-banner:hover {
+    cursor: pointer;
```

Comment:
Thanks for adding this, I made the banner changes live on stream very quickly when he hit 100% so didn't notice the lack of a tell that this element could be clicked :)